cli/azd/test/functional/testdata/recordings/Test_CLI_InfraBicepParam.docker.yaml

```diff
@@ -0,0 +1,3 @@
+version: "1.0"
```

Comment:
Will add a change https://github.com/Azure/azure-dev/pull/2741 to remove the noise

Reply:
@vhvb1989 should be able to merge and remove these